prediction_market_agent/agents/replicate_to_omen_agent/omen_resolve_replicated.py

```diff
@@ -114,18 +124,51 @@ def omen_finalize_and_resolve_and_claim_back_all_markets_based_on_others_tx(
         api_keys,
         created_finalized_markets,
     )
+    # Redeem from the resolved markets.
+    redeem_from_all_user_positions(api_keys)
     balances_after_resolution = get_balances(public_key)
     logger.info(f"{balances_after_resolution=}")
 
-    claimed = claim_all_bonds_on_reality(api_keys, finalized_before=now)
-
     return FinalizeAndResolveResult(
         finalized=finalized_markets,
         resolved=resolved_markets,
         claimed=claimed,
     )
 
 
+@observe()
+def find_resolution_on_other_markets_or_manually(
+    market: OmenMarket,
+    api_keys: APIKeys,
+) -> Resolution | None:
```

Comment:
I would rephrase, `manually` -> `using_ofv_resolver` or similar.
Manually at first glance means doing something outside of the script.

Reply:
True, renamed